crates/bevy_app/src/main_schedule.rs

```diff
@@ -142,13 +142,41 @@ impl Main {
         }
 
         world.resource_scope(|world, order: Mut<MainScheduleOrder>| {
+            let step = world
+                .get_resource_mut::<Stepping>()
```

Comment:
This should be documented on the method.

Reply:
I should have called out in the description that this is a draft PR / does not exist to be the final API.

Reply:
(not worth reviewing on that basis)

Reply:
Just illustrating and idea

Reply:
Updated the description!

Reply:
Ah got it.